TestRunner.psm1

```diff
@@ -561,21 +566,42 @@ function Invoke-TestRunner {
                         $_.InvocationInfo.PositionMessage,
                         $_.CategoryInfo.ToString(),
                         $_.FullyQualifiedErrorId
-                Write-InfoLog ($formatstring -f $fields)
+                Write-ErrorLog ($formatstring -f $fields)
             }
         }
 
         Write-InfoLog "Fetching new ODS log file content from $webServer"
         $logFile = Get-ConfigValue -Config $config -Key "PERF_API_LOG_FILE_PATH"
 
-        # Retreive API log file from the web server
+        Write-DebugLog "Log file path: $logFile" -LogLevel $logLevel
+
+        # Retrieve API log file from the web server
         if (Test-IsLocalhost $webServer) {
+            Write-DebugLog "Stopping IIS" -LogLevel $logLevel
+            Stop-Service W3SVC
```

Comment:
Running into error while API is running under IIS Express and don't have IIS feature turned on

Reply:
I'll have to think about how to avoid that.

Reply:
Please try the updated version, which attempts to detect if IIS exists.